azuredevops/internal/service/approvalsandchecks/resource_check_exclusive_lock.go

```diff
@@ -0,0 +1,41 @@
+package approvalsandchecks
+
+import (
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/microsoft/terraform-provider-azuredevops/azuredevops/internal/utils/converter"
+	"github.com/microsoft/terraform-provider-azuredevops/azuredevops/utils/pipelineschecksextras"
+)
+
+// ResourceCheckExclusiveLock schema and implementation for Exclusive Lock resource
+func ResourceCheckExclusiveLock() *schema.Resource {
+	r := genBaseCheckResource(flattenExclusiveLock, expandExclusiveLock)
+
+	r.Schema["timeout"] = &schema.Schema{
+		Type:     schema.TypeInt,
+		Optional: true,
+		Default:  43200,
```

Comment:
Add a validation `ValidateFunc: validation.IntBetween(1, 43200),` , the  value should between 1~43200